OPERATOR.rst

```diff
@@ -239,7 +239,7 @@ To do a promotion:
 
 #. Make sure your ``develop`` and ``prod`` branches are up to date.
 
-#. Check the ``prod`` branch for hotfixes. If there are changes on ``dev`` that
+#. Check the ``prod`` branch for hotfixes. If there are changes on ``develop`` that
```

Comment:
Unrelated to this PR but I wonder if we should use a commit tag to help us locate these PRs. Something like `[PH] - Permanent Hotfix`

Reply:
Good idea, Just `H` though for permanent and h for temporary hotfixes. Please create ticket for this, @amarjandu.